### PR TITLE
Update endpoints

### PR DESCRIPTION
**Pull request by neon-ninja**
Hi Konrad! Thanks for making this open source. This PR updates a few endpoints and gets the tests passing again.

### CODE REVIEW

File: tests/test_feed.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import Parler
 from Parler import with_auth as authed
 import os
+from pprint import pprint
```
Comment (KonradIT) [inline tests/test_feed.py]: Stray dependency.

**Review by KonradIT — CHANGES_REQUESTED** [review]

**Comment by KonradIT** [comment]
Thank you! Just a small nitpick but otherwise all good.

File: tests/test_feed.py (hunk 1 above)
Comment (neon-ninja) [reply tests/test_feed.py]: Fixed 👍

**Comment by KonradIT** [comment]
I can pass the tests with your changes locally but the CI goes haywire for some reason. Will investigate later.

**Comment by neon-ninja** [comment]
Looks like it's just the requests that require auth that failed on the CI - does the CI have valid PARLER_USERNAME and PARLER_PASSWORD environment variables? If so, maybe it's because the CI runs in Azure which has IP ranges that trigger Parler bot detection on login 🤔

**Comment by KonradIT** [comment]
That really sucks if true.

**Comment by KonradIT** [comment]
So weird, used to work before: https://github.com/KonradIT/parler-py-api/runs/5266815122?check_suite_focus=true

Merging anyways, thanks for keeping this up to date!!

**Pull request merged**